packages/components/src/components/form/BottomText.tsx

```diff
@@ -21,14 +23,6 @@ const slideDown = keyframes`
 `;
 
 export const Container = styled.div<{ $inputState?: InputState; $isDisabled?: boolean }>`
-    display: flex;
-    align-items: center;
-    gap: ${spacingsPx.xxs};
-    padding: ${spacingsPx.xs} ${spacingsPx.sm} 0 ${spacingsPx.sm};
-    min-height: ${BOTTOM_TEXT_MIN_HEIGHT}px;
-    color: ${({ $inputState, $isDisabled, theme }) =>
-        $isDisabled ? theme.textDisabled : getInputStateTextColor($inputState, theme)};
-    ${typography.label}
     animation: ${slideDown} 0.18s ease-in-out forwards;
 `;
```

Comment:
Is this used somewhere or is it a dead code?

Reply:
Oh, thanks for noticing - I forgot to use this.